test/utils/test_utils.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 var assert = require('assert');
-var TimeUuid = require('cassandra-uuid').TimeUuid;
+var TimeUuid = require('uuid').TimeUuid;
```

Comment:
The API of the uuid package is different.

Reply:
https://www.npmjs.com/package/uuid

Reply:
Replaced and corrected